pom.xml

```diff
@@ -94,11 +94,25 @@
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.0</version>
+        <version>3.3</version>
```

Comment:
FWIW, if you use annotation processing in those other projects, then you'll have to `mvn clean` before you can `mvn compile` again. This is a regression introduced in 3.2 that was fixed in 3.5.1. Look for “As an anecdote” in https://blog.ltgt.net/maven-is-broken-by-design-2/